spec/results-view-spec.js

```diff
@@ -701,6 +701,32 @@ describe('ResultsView', () => {
       atom.commands.dispatch(projectFindView.element, 'core:confirm');
       expect(resultsPane.refs.previewCount.textContent).toContain('Project search results');
     })
+  });
+
+  describe('search result context lines', () => {
+    beforeEach(async () => {
+      atom.config.set('find-and-replace.searchContextLineCountBefore', 2);
+      atom.config.set('find-and-replace.searchContextLineCountAfter', 3);
+      await atom.workspace.open('sample.js');
+
+      projectFindView.findEditor.setText('items');
+      atom.commands.dispatch(projectFindView.element, 'core:confirm');
+      await searchPromise;
+
+      resultsView = getResultsView();
+    });
+
+    it('shows the context lines', async () => {
+      if (parseFloat(atom.getVersion()) >= 1.17) {
+        const pathNode = resultsView.refs.listView.element.querySelectorAll(".search-result")[0];
+        expect(pathNode.querySelectorAll('.list-tree .list-item').length).toBe(5)
+        expect(pathNode.querySelectorAll('.list-tree .list-item')[0]).not.toHaveClass('match-line');
+        expect(pathNode.querySelectorAll('.list-tree .list-item')[1]).toHaveClass('match-line');
+        expect(pathNode.querySelectorAll('.list-tree .list-item')[2]).not.toHaveClass('match-line');
+        expect(pathNode.querySelectorAll('.list-tree .list-item')[3]).not.toHaveClass('match-line');
+        expect(pathNode.querySelectorAll('.list-tree .list-item')[4]).not.toHaveClass('match-line');
```

Comment:
Could you also assert about each list item's text content? With so many assertions, it might be good to extract the `querySelectorAll` expression into a variable, like `listItems` or something.